alfresco-transform-misc/alfresco-transform-misc/src/main/java/org/alfresco/transformer/transformers/StringExtractingContentTransformer.java

```diff
@@ -26,6 +26,9 @@
  */
 package org.alfresco.transformer.transformers;
 
+import org.slf4j.Logger;

```

Comment:
TODO: check logging across T-Core (and T-Service/T-Model) - needs to be consistent ?

Reply:
org.slf4j.Logger is the one used throughout T-Engines. I had to change code copied from the repo to use this one.

Reply:
Correction. org.apache.commons.logging.LogFactory was being used in this class already.
I made it consistent with the others.

Reply:
OK, thanks